src/H5FDsubfiling/H5FDioc_threads.c

```diff
@@ -744,15 +743,10 @@ ioc_file_queue_write_indep(sf_work_request_t *msg, int subfile_rank, int source,
     t_start       = MPI_Wtime();
     t_queue_delay = t_start - msg->start_time;
 
-#ifdef H5FD_IOC_DEBUG
-    if (sf_verbose_flag) {
-        if (sf_logfile) {
-            HDfprintf(sf_logfile,
-                      "[ioc(%d) %s]: msg from %d: datasize=%ld\toffset=%ld, "
-                      "queue_delay = %lf seconds\n",
-                      subfile_rank, __func__, source, data_size, file_offset, t_queue_delay);
-        }
-    }
+#ifdef H5_SUBFILING_DEBUG
+    H5_subfiling_log(file_context_id,
+                     "[ioc(%d) %s]: msg from %d: datasize=%ld\toffset=%ld, queue_delay = %lf seconds\n",
```

Comment:
Not super important for this PR, but I'd switch these to C99 format specifiers in the future and not use %ld for printing int64_t.